apps/s_server.c

```diff
@@ -2972,6 +2974,7 @@ static int www_body(int s, int stype, int prot, unsigned char *context)
     SSL *con;
     const SSL_CIPHER *c;
     BIO *io, *ssl_bio, *sbio;
+    BIO *bio = NULL;
```

Comment:
Please enclose this in `#ifdef CHARSET_EBCDIC` otherwise we get the compilation failure with  -Werror.